### ISSUE TITLE
stage accessibility - public or non-public

### ISSUE THREAD

**Issue by helenvarley** [upstage-org/limelight#242]
it should be possible to make a stage "public" - i.e. visible to someone who is not logged in, in the list of stages on the foyer page - or "non-public" - i.e. only visible to logged in players who have access to that stage.
it should be an option that can be set in the stage edit screen.

**Comment by rohrlach** [comment]
You mention "access to the stage"

Does this mean we need to create a permissions system?

**Comment by helenvarley** [comment]
yes ... would that be a separate issue, i guess?? ;)

**Comment by rohrlach** [comment]
Yes, #243 is necessary and should be done first.

But this is still valid.

**Comment by thongteav** [comment]
@helenvarley Just like to clarify a few points on this issue:
- On foyer page I've added a visibility column, so that a logged in user can see whether a stage is public or not. 
![image](https://user-images.githubusercontent.com/26214298/40092324-eba12c0c-5910-11e8-97a4-a098b533e1e1.png)
Audience won't see this column nor non-public stages:
![image](https://user-images.githubusercontent.com/26214298/40092378-33fd5688-5911-11e8-81c7-f94f835d6719.png)

- For assigning public or non-public stages, you can do it when you create a stage or in the stage edit page:
![image](https://user-images.githubusercontent.com/26214298/40092474-8e6aaf1c-5911-11e8-994a-4bcdc1fd9aff.png)
![image](https://user-images.githubusercontent.com/26214298/40092453-7cd65b16-5911-11e8-9e8c-60786c476c29.png)
- Restriction to the stages will be done in #243 

Is there anywhere else you'd like the player to see whether a stage is public or not. Maybe in the stages list page or each stage index page?

**Comment by rohrlach** [comment]
The access column could show for audience - and just say that access level is audience?

**Comment by helenvarley** [comment]
@thongteav yes it makes sense to include the public/non-public status in the stages list page, as well as on each stage edit page.

@rohrlach - would audience be confused, seeing a column called "access" that always has "audience" in it? is it difficult to hide this column from the audience?

**Comment by rohrlach** [comment]
Sure - I was just thinking about how it is now.  But sure, my preference would be hide useless information wherever possible 😄

**Comment by thongteav** [comment]
- Added an 'accessibility' column in the stage list page:
![image](https://user-images.githubusercontent.com/26214298/40395672-53069cd0-5e7d-11e8-91ac-f41bf0061547.png)
